cmd/gaia/contrib/gitian-build.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -euo pipefail
+
+g_default_dirname="gitian-build-$(date +%Y-%m-%d-%H%M%S)"
+g_dirname=''
+
+f_main() {
+  local l_dirname \
+    l_workdir \
+    l_sdk \
+    l_commit \
+    l_platform \
+    l_result \
+    l_descriptor \
+    l_release_name
+
+  l_dirname=$1
+  l_platform=$2
+  l_sdk=$3
+  pushd ${l_sdk}
+  l_commit=$(git rev-parse HEAD)
+  popd
+
+  l_descriptor=$l_sdk/cmd/gaia/contrib/gitian-descriptors/gitian-${l_platform}.yml
+  [ -f ${l_descriptor} ]
+
+  l_workdir="$(pwd)/${l_dirname}"
+  mkdir ${l_workdir}/
+  echo "Download gitian" >&2
+  git clone https://github.com/devrandom/gitian-builder ${l_workdir}
+
+  echo "Prepare gitian-target docker image" >&2
+  f_prep_docker_image "${l_workdir}"
+
+  echo "Download Go" >&2
+  f_download_go "${l_workdir}"
+
+  echo "Start the build" >&2
+  f_build "${l_workdir}" "${l_sdk}" "${l_commit}" "${l_platform}"
+  echo "You may find the result in $(echo ${l_workdir}/result/*.yml))" >&2
+
+  l_release_name="$(sed -n 's/^name: \"\(.\+\)\"$/\1/p' ${l_descriptor})"
+  [ -n ${l_release_name} ]
+  echo "You can now sign the build with the following command:" >&2
+  echo "cd ${l_workdir} ; bin/gsign -p 'gpg --detach-sign --armor' -s GPG_IDENTITY --release=${l_release_name} ${l_descriptor}" >&2
```

Comment:
I've got the `l_release_name` variable empty in this helper during my tests

Reply:
So it actually failed at line 44, didn't it?

Reply:
Actually it went through and printed the helper of line 46, but empty.
With my Ubuntu laptop instead I had this variable filled (tried this morning at home)

Reply:
fixed